src/RadhydroSimulation.hpp

```diff
@@ -94,6 +95,8 @@ template <typename problem_t> class RadhydroSimulation : public AMRSimulation<pr
 	SimulationData<problem_t> userData_;
 
 	int enableCooling_ = 0;
+	int enableChemistry_ = 0;
+	Real max_density_allowed = NAN;
```

Comment:
This should probably be set to `std::numeric_limits<amrex::Real>::max()`

Reply:
So, like this: `Real max_density_allowed = std::numeric_limits<amrex::Real>::max()` ?